packages/forms/src/UIForm/fieldsets/Array/Array.component.js

```diff
@@ -168,9 +165,8 @@ ArrayWidget.defaultProps = {
 
 if (process.env.NODE_ENV !== 'production') {
 	ArrayWidget.propTypes = {
-		errorMessage: PropTypes.string,
+		displayMode: PropTypes.string,
```

Comment:
required ?

Reply:
No. If not provided, it will fallback to classical widget id